src/fe-small-advertisements/src/actions/types.ts

```diff
@@ -0,0 +1,3 @@
+import { ChangeNameAction } from './name';
+
+export type UserAction = ChangeNameAction;
```

Comment:
I dont het this? Why weneed to have a `ChangeNameAction` _and_ a `UserAction` if they both the same?

Reply:
I was trying to set the basis to in the future do something like: 

`export type UserAction = ChangeNameAction | ChangeAgeAction | ChangePasswordAction` 

I was just practicing the structure, but I realize it is not good to write code for no reason, my bad.

 To learn Typescript in React and how to use it with Redux  is confusing  sometimes :). Thanks for guidance!

Reply:
You will be fine. Keep the good work up dude. You doind fantasic